README.md

```diff
@@ -138,6 +138,7 @@ The following options are available with all commands. You must use command line
 
 - `--url, -u "protocol://host:port/dbname"` - specify the database url directly. _(env: `DATABASE_URL`)_
 - `--env, -e "DATABASE_URL"` - specify an environment variable to read the database connection URL from.
+- `--env-file ".env"` - specify an alternate environment variables file(s) to load.
```

Comment:
might be helpful to give an example of how to specify multiple files, and mention which files are loaded automatically

Reply:
Agreed, there should be an "Environment Variables" section in the README that explains how dbmate uses them, how `.env` is loaded automatically, and variable precedence: dot-env files, calling process environment, command line arguments.

I created issue #500 for it.  I'll write it up if no one else volunteers, but it's a good opportunity for someone else to contribute, so it's probably okay to leave it up for a while.